tests/support_server/setup.pm

```diff
@@ -283,6 +280,100 @@ sub setup_xdmcp_server {
     $xdmcp_server_set = 1;
 }
 
+sub setup_iscsi_server {
+    return if $iscsi_server_set;
+
+    # If no LUN number is specified we must die!
+    my $num_luns = get_var("NUMLUNS");
+    die "no LUN number defined, NUMLUNS variable missing?" unless $num_luns;
+
+    # A second disk for iSCSI LUN is mandatory
+    my $hdd_lun_size = get_var("HDDSIZEGB_2");
+    die "not enough space for iSCSI LUN, HDDSIZEGB_2 variable missing?" unless $hdd_lun_size;
+
+    # Integer part of the LUN size is keep and can't be lesser than 1GB
+    my $lun_size = int($hdd_lun_size / $num_luns);
+    die "iSCSI LUN cannot be lesser than 1GB!" if ($lun_size < 1);
+
+    # Are we using virtio or virtio-scsi?
+    # Note: it should be possible to do this in pure Perl code...
```

Comment:
I'm not a Perl expert, so it could take more time...
For a next version maybe, if this code could be accepted this time :-)